runtime/core/websocket/websocket_server.cc

```diff
@@ -102,7 +108,8 @@ void ConnectionHandler::OnSpeechData(const beast::flat_buffer& buffer) {
 std::string ConnectionHandler::SerializeResult(bool finish) {
   json::array nbest;
   for (const DecodeResult& path : decoder_->result()) {
-    json::object jpath({{"sentence", path.sentence}});
+    std::string sentence = post_processor_->Process(path.sentence, finish);
+    json::object jpath({{"sentence", sentence}});
     if (finish) {
       json::array word_pieces;
       for (const WordPiece& word_piece : path.word_pieces) {
```

Comment:
post_processor_->Process(string, finish) will be called only in function `ConnectionHandler::SerializeResult`, it wil do:

1.  process blank (according to configuration)
2.  itn / punc (according to finish tag)